Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision d915e1522d4402b38eb9cbf867614f05f42c4c24)
+++ b/.gitignore	(date 1673690434451)
@@ -1,1 +1,2 @@
 
+.idea
\ No newline at end of file
Index: .github/workflow/my_first_workflow.yaml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.github/workflow/my_first_workflow.yaml b/.github/workflow/my_first_workflow.yaml
new file mode 100644
--- /dev/null	(date 1673690426971)
+++ b/.github/workflow/my_first_workflow.yaml	(date 1673690426971)
@@ -0,0 +1,14 @@
+name: GitHub Actions Demo
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - run: echo "Trigerem jest ${{ github.event_name }} event."
+      - run: echo "Job dziala na systemie ${{ runner.os }} by GitHub!"
+      - run: echo "Twoj branch ${{ github.ref }} , repozytorium ${{ github.repository }}."
\ No newline at end of file
